### PR TITLE
Rebasing on dev to get MockPointer changes for #3375

### PR DESCRIPTION
**Pull request by starakaj**
Basically making a new PR for #3375 so that it can pick up MockPointer changes

### CODE REVIEW

**Comment by starakaj** [comment]
@bigtimebuddy does this work?

**Comment by bigtimebuddy** [comment]
Looks like it's still failing on these tests: https://travis-ci.org/pixijs/pixi.js/builds/189202921#L639-L662 can you take a look at that. Maybe @staff0rd can help.

**Comment by starakaj** [comment]
Yeah, it looks like some tests got added that have a different spec from the changes I made. I'll look at this today.

**Comment by staff0rd** [comment]
I forked @starakaj's branch, and tried to make a PR, but result looks like i'm trying to push all commits since  his last rebase into the branch also.  Not sure what I should do here, the fix for the tests is in this commit;

https://github.com/staff0rd/pixi.js/commit/f71055e478caf1a364645eee189e0aecc31b2505

**Comment by bigtimebuddy** [comment]
@starakaj could you incorporate @staff0rd's fix? We would really like to get this into our 4.4 release. Thanks for all your help.

**Comment by starakaj** [comment]
Yes, sorry, I can do that today. Happy to help!

**Comment by starakaj** [comment]
Cool, let me know if you need anything else!

File: src/interaction/interactiveTarget.js [hunk 1]
```diff
@@ -54,52 +54,16 @@ export default {
...
-
-    /**
-     * Internal check to detect if the pointer cursor is hovered over the displayObject
-     *
-     * @inner {boolean}
+     * @returns {Map<number, InteractionTrackingData>} Map of all tracked pointers, by identifier
      * @private
      */
-    _pointerOver: false,
+    getTrackedPointers: function getTrackedPointers()
```
Comment (starakaj) [inline src/interaction/interactiveTarget.js]: I believe so!

**Pull request merged**